gateway/proxy_fil.go

```diff
@@ -441,13 +441,23 @@ func (gw *Node) GetActorEvents(ctx context.Context, filter *types.ActorEventFilt
 	if err := gw.limit(ctx, stateRateLimitTokens); err != nil {
 		return nil, err
 	}
+	if filter != nil && filter.FromHeight != nil {
```

Comment:
I guess "filter == nil" means "current tipset"?

Reply:
yeah, it ends up being the same as `filter.FromHeight == nil` because we just insert a zero value filter in place; do we need to account for the "last finalized" tipset in this limiting? can you have a lookback limit of 0?

Reply:
I think we can leave it as-is.